object/service.proto

```diff
@@ -262,7 +262,17 @@ service ObjectService {
   rpc Replicate(ReplicateRequest) returns (ReplicateResponse);
 }
 
-// GET object request
+// GET object request.
+//
+// The query for a parent object's EC part locally stored on the server is
+// specified as follows:
+//  - `body.address` is an address of the parent;
+//  - `meta_header.x_headers` includes `__NEOFS__EC_RULE_IDX` and
+//    `__NEOFS__EC_PART_IDX` by object attribute format. Rule index MUST NOT
```

Comment:
What if netmap change happens and we're requesting part 0/4 from node that stores 0/1?